### DISCUSSION TITLE
Tailwind Components

### DISCUSSION THREAD

**Discussion by himeshp** [Etheonor/supanextail#28] [Feedback]
Using ready-to-use Tailwind components would give a better feel of the UI.

https://tailwinduikit.com/

**Comment by Etheonor** [comment]
I'm not a fan of using pre built components in a boilerplate, as the user will certainly use something else. I have already integrated DaisyUI which is a bit more flexible than a pre-built component, and has the advantage of being free.

Maybe improve the already existing UI is something to consider (it's clearly not my best area of expertise)?

**Reply by himeshp** [reply]
yes, I understand, I guess it makes more sense.
but the UI can be more appealing for sure If you check other competitors out there like jumpstart pro for example.

**Reply by Etheonor** [reply]
I'll try to rework the landing page asap!

**Reply by Etheonor** [reply]
I just checked this article and maybe there are better plugins for that. Don't know if you have a preference but I may need to rework and implement something else, don't know yet! https://dev.to/nikit/15-tailwindcss-component-libraries-that-will-make-your-life-a-bit-easier-15eg

**Reply by himeshp** [reply]
Yes, there are many, you might want to check license and terms of usage first

**Comment by Etheonor** [comment]
I'm working on a new design, don't hesitate to tell me if it's better or not, and what you want to see 
![image](https://user-images.githubusercontent.com/35852770/126846171-ecb9ab06-ed42-42c2-a461-bd71b8203a64.png)

**Reply by Etheonor** [reply]
I created an Adobe XD link, don't hesitate to tell me what you think https://xd.adobe.com/view/9e54329f-c4b2-4e62-9baf-654402b6ff29-cb07/

**Reply by himeshp** [reply]
yes, this is much clean and neat
